### ISSUE TITLE
Old `align="center"` bad fixes

### ISSUE THREAD

**Issue by GeoffreyParrier** [vuetifyjs/eslint-plugin-vuetify#121]
**Here's the problem:**
> Source
```vue
<v-col align="center" />
```

> Incorrect fixes (current)
```vue
<v-col classes="align" />
```

> Correct fixes
```vue
<v-col classes="align-center" />
```
